README.md

```diff
@@ -1,69 +1,110 @@
+project-specific adaptations.
 
-## Installation
-
-Prerequisites:
+# Requirements
 
 * [Installed shared library][piper-library-installation] of project "Piper".
 * Jenkins User with privileges to create jobs.
 
-There are two recommended ways to consume our `Jenkinsfile`. For both ways, you need to create a new pipeline job in Jenkins that [retrieves a pipeline from an SCM][jenkins-doc-pipelineFromSCM].
-The location of the `Jenkinsfile` may differ:
+# Download and Installation
+
+There are two recommended ways to consume our `Jenkinsfile`. For both ways, you
+need to create a new pipeline job in Jenkins that [retrieves a pipeline from an
+SCM][jenkins-doc-pipelineFromSCM]. The location of the `Jenkinsfile` may
+differ:
+
+1. The `Jenkinsfile` can be copied into the source code repository of the
+   application. In that case, no additional pipeline job parameters are required
+   to retrieve the application source code repository.
+
+2. The `Jenkinsfile` can be kept in its own repository separate from the
+   application. In case you want to apply the `Jenkinsfile` to multiple
+   applications, this approach is beneficial since you only need to maintain the
+   `Jenkinsfile` in one place. You can consume the `Jenkinsfile` e.g. directly
+   from this repository on GitHub.com or from your own fork of it. You must
+   define two mandatory Jenkins pipeline job parameters to specify the location
+   of your application:
 
-1. The `Jenkinsfile` can be copied into the source code repository of the application. In that case, no additional pipeline job parameters are required to retrieve the application source code repository.
+    * `GIT_URL`: The URL to the Git repository that the application resides in.
+    * `GIT_BRANCH`: The branch of the Git repository that should be checked out
+      for the build.
 
-2. The `Jenkinsfile` can be kept in an own repository separate from the application. In case you want to apply the `Jenkinsfile` to multiple applications, this approach is beneficial since you only need to maintain the `Jenkinsfile` in one place. You must define two mandatory Jenkins pipeline job parameters in this case:
+Example configuration for consumption of a `Jenkinsfile` directly out of this
+repository on GitHub.com: 
 
-    * `GIT_URL`: The URL to the Git repository in which the application resides.
-    * `GIT_BRANCH`: The branch of the Git repository that should be checked out for the built.
+![Library Setup](./documentation/docs/images/setupInJenkins.png)
 
-Each `Jenkinsfile` requires project specific configuration files in the application sources. Templates for these configuration files are available in the same folders as the respective `Jenkinsfile`.
+# Configuration
+
+Each `Jenkinsfile` requires project specific configuration files in the
+application sources. Templates for these configuration files are available in
+the same folders as the respective `Jenkinsfile`.
 
 ## Overview of available pipelines
```

Comment:
I'd rather see this under `# What you get` that here in `# Configuration` . Any reason why this is here?

Reply:
Correct. Moved upwards.